src/main/java/com/mongodb/kafka/connect/source/MongoSourceTask.java

```diff
@@ -158,7 +160,8 @@ public void start(final Map<String, String> props) {
     long nextUpdate = startPoll + sourceConfig.getLong(POLL_AWAIT_TIME_MS_CONFIG);
     String prefix = sourceConfig.getString(TOPIC_PREFIX_CONFIG);
     Map<String, Object> partition = createPartitionMap(sourceConfig);
-
+    JsonWriterSettings jsonOptions = null;
+    jsonOptions = handleJsonOptions();
```

Comment:
Join declaration and assignment:
```suggestion
    JsonWriterSettings jsonOptions = handleJsonOptions();

```